content/en/blog/_posts/2021-11-23-object-storage-support.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: 'Introducing COSI: Object Storage Support in Kubernetes'
+date: 2021-11-23
```

Comment:
```suggestion
date: 2021-12-14
```

Reply:
Remember also to change the path (`content/en/blog/_posts/2021-11-23-object-storage-support.md`).